composer.json

```diff
@@ -39,7 +39,8 @@
     "require-dev": {
         "fzaninotto/faker": "^1.5",
         "graham-campbell/testbench-core": "^1.0",
-        "phpunit/phpunit": "^4.7.6"
+        "phpunit/phpunit": "^4.7.6",
+        "laravel/homestead": "dev-master"
```

Comment:
This isn't right. This is meant to be a global install so should not go in here. Also, never use dev-master anyway.